libs/eo/transfers/src/lib/eo-transfers.component.ts

```diff
@@ -39,11 +47,22 @@ import { EoTransfersStore } from './eo-transfers.store';
       </p>
     </watt-card>
     <watt-card class="watt-space-stack-m">
-      <eo-transfers-table></eo-transfers-table>
+      <eo-transfers-table
+        [transfers]="transfers$ | push"
+        [selectedTransfer]="selectedTransfer$ | push"
+        (transferSelected)="store.setSelectedTransfer($event)"
+      ></eo-transfers-table>
     </watt-card>
   `,
 })
-export class EoTransfersComponent {
+export class EoTransfersComponent implements OnInit {
   error$ = this.store.error$;
-  constructor(private store: EoTransfersStore) {}
+  transfers$ = this.store.transfers$;
+  selectedTransfer$ = this.store.selectedTransfer$;
+
+  constructor(protected store: EoTransfersStore) {}
+
+  ngOnInit(): void {
+    this.store.getTransfers();
```

Comment:
Question: why use the OnInit instead of placing it in the constructor? :)

Reply:
The reason was to use the "inject" pattern instead, which should be done now. Besides that, it won't make any difference in the current state of the application, but in the future ex. Implementing transitions between the pages could have, and also, it's a common practice not to have logic inside the constructor.